torch/csrc/api/src/nn/modules/normalization.cpp

```diff
@@ -0,0 +1,40 @@
+#include <torch/nn/modules/normalization.h>
+
+#include <torch/cuda.h>
+#include <torch/types.h>
+#include <torch/utils.h>
+#include <torch/nn/init.h>
+
+#include <ostream>
+#include <utility>
+
+namespace torch {
+namespace nn {
+
+LayerNormImpl::LayerNormImpl(const LayerNormOptions& options_) : options(options_) {
+  reset();
+}
+
+void LayerNormImpl::reset() {
+  if (options.elementwise_affine()) {
+    weight = register_parameter("weight", torch::ones(torch::IntArrayRef(options.normalized_shape())));
+    bias = register_parameter("bias", torch::zeros(torch::IntArrayRef(options.normalized_shape())));
```

Comment:
similarly here, we should be able to remove `torch::IntArrayRef`.

Reply:
similarly here, we should probably use `torch::empty`. And do `torch::nn::init::zeros_(bias)` at the end of `reset()`.